web-app/js/portal/lang/en.js

```diff
@@ -149,6 +149,8 @@ OpenLayers.Lang.en = OpenLayers.Util.extend(OpenLayers.Lang.en, {
     btnSave: 'Save',
     btnCancel: 'Cancel',
 
+    errorLoadingCollection: 'Error loading collection with UUID \'{0}\'',
```

Comment:
@danfruehauf , I think the message should be more specific - i.e. indicate that the cause of the error is an invalid UUID.  'The data collection cannot be loaded because the UUID \'{0}\' does not exist'.